_includes/sass/my-bootstrap.scss

```diff
@@ -8,6 +8,19 @@
 $font-size-base: 1.1rem;
 $paragraph-margin-bottom: 1.5em;
 
+/* Add xxxl breakpoint */
+$grid-breakpoints: ( xs: 0, sm: 576px, md: 768px, lg: 992px, xl: 1200px, xxl:
+1400px , xxxl: 1680);
+
+/* Add size definition for container in xxxl breakpoint */
+$container-max-widths: (
```

Comment:
Maybe use `map-merge` to avoid redefining other values?

Reply:
The other way around, but bootstrap doesn't use map-merge for `$container-max-widths`